src/SFA.DAS.ApprenticeCommitments.Web/TagHelpers/ConfirmationHelper.cs

```diff
@@ -18,16 +18,22 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
         }
     }
 
-    [HtmlTargetElement("form", Attributes = "confirmation-form")]
-    public class YesNoFormHelper : TagHelper
+
+    [HtmlTargetElement("yes-no-wrapper")]
+    public class YesNoWrapHelper : TagHelper
     {
         [ViewContext]
         public ViewContext ViewContext { get; set; } = null!;
-
         public override async Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
         {
+            output.TagName = "div";
+            output.Attributes.Add("class", "govuk-form-group");
+            output.PreContent.SetHtmlContent(@"<fieldset class=""govuk-fieldset"">");
+            output.PostContent.SetHtmlContent("</fieldset>");
+            
             if (ViewContext?.ModelState.IsValid == false)
-                output.Attributes.Add("class", "govuk-form-group--error");
+                output.Attributes.RemoveAll("class");
+                output.Attributes.Add("class", "govuk-form-group govuk-form-group--error");
```

Comment:
I think there are braces missing here - assuming both of these statements fall inside the `if`?

Reply:
Thanks, fixed